htmy/renderer/context.py

```
@@ -0,0 +1,27 @@
+        Returns the renderer instance from the given context.
+
+        Arguments:
+            context: The context the renderer instance should be loaded from.
+
+        Raises:
+            KeyError: If no renderer instance was found in the context.
+        """
+        renderer = context[cls]
+
+        if not is_renderer(renderer):
+            raise TypeError("The value corresponding to RendererContext in the context must be a renderer.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)